### PR TITLE
build: replace bower by yarn

### PR DESCRIPTION
**Pull request by marie-j**
# Replace bower by yarn

Target branch is correct as other changes are to come and it will ease QC testing

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,7 @@
 #### SYSTEM COMMAND ####
 NODE=node
-NPM=npm
+NPM=yarn
```
Comment (FredericEspiau) [inline Makefile]: Couldn't you also rename the variable name ? 😄

File: Makefile [hunk 2]
```diff
@@ -22,9 +20,9 @@ GRUNT_DEP=$(NODE_DIR)/grunt
 VERSION=`grep -Po '(?<="version": ")[^"]*' package.json`
 
 #### OTHER ####
-ifneq ($(strip $(bower_registry)),)
-BOWER_PARAM=--config.registry=$(bower_registry)
-endif
+# ifneq ($(strip $(bower_registry)),)
```
Comment (FredericEspiau) [inline Makefile]: If it's unused, it can be safely removed :)

File: package.json [hunk 3]
```diff
@@ -11,8 +11,13 @@
   "author": "OVH SAS",
   "main": "./dist/ovh-angular-apiv7.min.js",
   "repository": "ovh-ux/ovh-angular-apiv7",
-  "dependencies": {},
+  "dependencies": {
```
Comment (FredericEspiau) [inline package.json]: Wasn't there a reference to bower somewhere here ?

**Comment by antleblanc** [comment]
We will handle the change by upgrading to the shared rollup configuration.

**Pull request closed without merging**